pkg/virt-api/webhooks/mutating-webhook/mutators/migration-create-mutator.go

```diff
@@ -58,11 +58,6 @@ func (mutator *MigrationCreateMutator) Mutate(ar *admissionv1.AdmissionReview) *
 	addMigrationFinalizer(&migration)
 
 	patchBytes, err := patch.GeneratePatchPayload(
-		patch.PatchOperation{
```

Comment:
It looks like a more accurate "fix" is to patch the specific entries and not all the metadata.

Reply:
I agree that patching the labels and finalizers are a better way to go.
The removal of the replacement of the spec was done in this PR for two reasons:
1. It is redundant.
2. It simplifies the tests.

I will create a follow-up PR to return more precise patches.

Reply:
I am missing the reason you cannot fix it directly now.
You change the patch already, so changing it in a more precise manner is a fit IMO.

But I will not insist.